README.md

```diff
@@ -25,14 +25,24 @@ Manticore is a symbolic execution tool for analysis of binaries and smart contra
 
 Manticore can analyze the following types of programs:
 
-- Linux ELF binaries (x86, x86_64 and ARMv7)
 - Ethereum smart contracts (EVM bytecode)
+- Linux ELF binaries (x86, x86_64 and ARMv7)
 
 ## Usage
 
 ### CLI
 
-Manticore has a command line interface which can be used to easily symbolically execute a supported program. Analysis results will be placed into a new directory beginning with `mcore_`.
+Manticore has a command line interface which can be used to easily symbolically execute a supported program or smart contract. Analysis results will be placed into a new directory beginning with `mcore_`.
+
+Use the CLI to explore possible states in Ethereum smart contracts. Manticore includes _detectors_ which flag certain conditions, including known vulnerable code, as it explores possible states. Solidity smart contracts must have a `.sol` extension for analysis by Manticore. See a [demo](https://asciinema.org/a/154012).
```

Comment:
i know was just copied and pasted, but while we're here... i think "which flag certain conditions, including known vulnerable code" sounds kind of weird ( i think i wrote it :x ).

maybe this?

```
Manticore includes _detectors_ which flag potentially vulnerable code that is detected in an explored state.
```